js/scripts.js

```diff
@@ -0,0 +1,67 @@
+// Testing if page width is mobile size.
+function testPageWidth() {
+  const breakPoint = 680,
+        treeContainer = document.getElementById("tree"),
+        trees = document.getElementsByClassName("hierplane");
+
+  // If desktop screen width:
+  if (window.innerWidth >= breakPoint) {
+    // If no trees exist
+    if (trees.length === 0 && pageId === "home") {
+      // Initialize Hierplane with data (see treeData.js)
+      hierplane.renderTree(introData, { target: "#tree" });
+    }
+    // If #tree container has mobile class
+    if (tree.classList.contains("mobile")) {
+      // Remove mobile class
+      treeContainer.classList.remove("mobile");
+    }
+  // Otherwise, if mobile screen width:
+  } else {
+    // Loop through .hierplane instances
+    while(trees[0]) {
+      // Remove each .hierplane
+      trees[0].parentNode.removeChild(trees[0]);
+    }
+    // Add mobile class
+    treeContainer.classList.add("mobile");
+  }
+}
```

Comment:
Woah, this is crazy.  What are we doing here, and why?

Reply:
See top-level comment about what I was doing w/ mobile resize stuff.

Reply:
Removed the JS mobile stuff in latest commit.